MIGRATION_GUIDE.md

```diff
@@ -3,6 +3,41 @@
 
 This guide outlines how developers can migrate from older versions of our SDK to newer ones.
 
+## Migrating to v4.0.0
+
+### `Event.EventName` enum:
+
+- We have removed the PascalCase names in the `Event.EventName` enum, leaving only the camelCase names. You will need to remove any references to the old PascalCase names.
+- We have removed the `.OpenedPush` case (there is no camelCase replacement), as this is intended for internal use only. You will need to remove any references to the `.OpenedPush` case.
+
+## Migrating to v3.3.0
```

Comment:
wasn't this merged to master? I updated as/rel-400 with the changes from master after you merged. Sorry I should have let you know.